static/styles/subscriptions.css

```diff
@@ -961,27 +961,43 @@ h4.user_group_setting_subsection_title {
     vertical-align: middle;
 }
 
-#stream_privacy_modal,
-.stream-creation-body {
-    .input-group {
-        margin-bottom: 10px;
+div.settings-radio-input {
+    border-bottom: 1px solid hsl(0, 0%, 87%);
+    margin: 2px 0 2px 5px;
+    padding: 2px 0;
 
-        .radio-input-parent {
-            border-bottom: 1px solid hsl(0, 0%, 87%);
-            margin: 2px 0 2px 5px;
-            padding: 2px 0;
+    &:last-of-type {
+        border-bottom: none;
+    }
 
-            &:last-of-type {
-                border-bottom: none;
-            }
+    label.radio {
+        display: inline;
+        margin: 5px;
+        padding-left: 0px;
+    }
 
-            label.radio {
-                display: inline;
-                margin: 5px;
-                padding-left: 0;
-            }
+    input {
```

Comment:
Note that `margin` for this `input` is defined separately, since there are different margins for stream creation and edit form. Though this will no longer be required when we remove the modal in stream edit panel as per #23013.

Reply:
Cool, let's try to remember to do that when we merge that PR.